### PR TITLE
New IA - Fundamentals (Node)

### PR DESCRIPTION
**Pull request by nlarew**
### Issue JIRA link:

https://jira.mongodb.org/browse/DOCSP-13773

### Docs staging link (requires sign-in on MongoDB Corp SSO):

- [Realms](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/new-ia-fundamentals/sdk/node/fundamentals/realms)
- [Realm Objects](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/new-ia-fundamentals/sdk/node/fundamentals/realm-objects)
- Write Transactions (WIP)
- Live Queries (WIP)
- Migrations (WIP)
- MongoDB Realm Application Services (WIP)

### CODE REVIEW

File: source/sdk/node/fundamentals/live-queries.txt [hunk 1]
```diff
@@ -11,3 +12,136 @@ Live Queries
    :backlinks: none
    :depth: 2
    :class: singlecol
+
+Live Queries
+------------
+
+You can, of course, easily read back the data that you have
```
Comment (cbush) [inline source/sdk/node/fundamentals/live-queries.txt:19]: I wrote this originally but have changed my mind.
```suggestion
You can read back the data that you have
```

**Review by cbush — APPROVED** [review]
LGTM!

File: source/sdk/node/fundamentals/live-queries.txt [hunk 2]
```diff
@@ -11,3 +12,136 @@ Live Queries
...
+your application to slow down.
+
+{+client-database+} bypasses all of this using :term:`zero-copy`
+:term:`live objects`. :term:`{+service-short+} object` accessors point directly into
+database storage using memory mapping, so there is no distinction
+between the objects in {+client-database+} and the results of your query in
+application memory. Because of this, you can traverse direct references
+across an entire {+realm+} from any query result.
+
+Write Transactions
```
Comment (cbush) [inline source/sdk/node/fundamentals/live-queries.txt:146]:
```suggestion
```

File: source/sdk/node/fundamentals/live-queries.txt [hunk 3]
```diff
@@ -11,3 +12,136 @@ Live Queries
...
+------------------
```
Comment (cbush) [inline source/sdk/node/fundamentals/live-queries.txt:147]:
```suggestion
```

**Comment by nlarew** [comment]
Gonna hold off on the live-queries suggestions - those page are mainly just holding old content for now and will be updated in the next PR.

**Pull request merged**